lexicons/tools/ozone/set/remove.json

```diff
@@ -0,0 +1,30 @@
+{
+  "lexicon": 1,
+  "id": "tools.ozone.sets.remove",
+  "defs": {
+    "main": {
+      "type": "procedure",
+      "description": "Remove values from a specific set",
```

Comment:
should describe semantics here in the description: if you try to remove a string which isn't in the set, is that an error? (I think not, but double-checking, and should document it here)